lms/djangoapps/instructor/tests/test_services.py

```diff
@@ -147,9 +147,10 @@ def test_report_suspicious_attempt(self):
         """
         requester_name = "edx-proctoring"
         email = "edx-proctoring@edx.org"
-        subject = "Proctored Exam Review: {review_status}".format(review_status="Suspicious")
+        subject = u"Proctored Exam Review: {review_status}".format(review_status="Suspicious")
```

Comment:
Is there a test for when `review_url` is not None? It might make sense to add that to make sure that the URL is being included in the e-mail message.

Reply:
I've added that case to the test.

I ran `npm install`, but the eslint errors keep popping up.